app/controllers/hearings_controller.rb

```diff
@@ -61,18 +61,19 @@ def set_hearing_events
   def hearing
     logger.info 'USING_V1_ENDPOINT'
     @hearing ||= helpers.decorate(@prosecution_case.hearings.find do |hearing|
-                                    hearing.id == params[:id]
+                                    hearing.id == hearing_id
                                   end, HearingDecorator)
   end
 
   def hearing_v2_call
     @hearing ||= helpers.decorate(
-      CdApi::Hearing.find(params[:id], params: {
+      CdApi::Hearing.find(hearing_id, params: {
                             date: paginator.current_item.hearing_date.strftime('%F')
                           }), CdApi::HearingDecorator
     )
   rescue ActiveResource::ResourceNotFound
-    redirect_to_prosecution_case(notice: I18n.t('hearings.show.flash.notice.no_hearing_details'))
+    # Return empty hearing so we can still display the page
```

Comment:
Is it worth monitoring this on Sentry with `Sentry.capture_exception(e)` ?

Reply:
This isnt an exception as such so no need to capture in Sentry. The hearing summaries can show hearings that were never hearings (things that were scheduled but never heards, future hearings etc) so there is a valid reason for them being on the summaries but their not being a hearing for them.